lib/services/user_category_service.dart

```diff
@@ -0,0 +1,7 @@
+class UserCategoryService {
+  Future<List> getCategories() async {
+    // List of another 2 dummy questions (different)
+    List<String> userCategories = ["Lifestyle", "Technology"];
+    return userCategories;
+  }
+}
```

Comment:
should we leave the categories as lists or as boxes like shown on Figma?

Reply:
Due to how a user can have varying numbers of length for categories, there is no real clean way to create the dynamic rows and columns that I know of. If you know of a way of making flutter create a new row when there is already two in that how, please let me know.